cmd/vic-machine/inspect/inspect.go

```diff
@@ -144,7 +144,7 @@ func (i *Inspect) Run(clic *cli.Context) (err error) {
 		return errors.New("inspect failed")
 	}
 
-	vchConfig, err := executor.GetVCHConfig(vch)
+	vchConfig, err := executor.GetNoSecretVCHConfig(vch)
```

Comment:
Thinking about this, I would have one method to get VCH config where the second option would be a secret. If the secret is empty, I would treat it as no secret. Just saying.